src/Arcus.Messaging.Tests.Integration/appsettings.json

```diff
@@ -15,6 +15,10 @@
       }
     },
     "ServiceBus": {
+      "Docker": {
+        "ConnectionStringWithQueue": "#{Arcus.ServiceBus.Docker.ConnectionStringWithQueue}#",
+        "ConnectionStringWithTopic": "#{Arcus.ServiceBus.Docker.ConnectionStringWithTopic}#" 
+      },
       "ConnectionStringWithQueue": "#{Arcus.ServiceBus.ConnectionStringWithQueue}#",
```

Comment:
Nit: You might want to move this to a subcategory then as well for consistency reasons

Reply:
OK, added sub-category `SelfContained`. Resolving.